source/validate_id.cpp

```diff
@@ -2002,11 +2002,52 @@ bool idUsage::isValid<OpBranch>(const spv_instruction_t *inst,
                                 const spv_opcode_desc opcodeEntry) {}
 #endif
 
-#if 0
 template <>
-bool idUsage::isValid<OpBranchConditional>(
-    const spv_instruction_t *inst, const spv_opcode_desc opcodeEntry) {}
-#endif
+bool idUsage::isValid<SpvOpBranchConditional>(
+    const spv_instruction_t *inst, const spv_opcode_desc)
+{
+  const size_t numOperands = inst->words.size() - 1;
+  constexpr size_t condOperandIndex = 1;
```

Comment:
Consider const. Microsoft doesn't fully support the C++ standard.
https://msdn.microsoft.com/en-us/library/hh567368.aspx

Reply:
Yup.  We support VS 2013.  Sadly, constexpr can't be used in SPIRV-Tools.
Use const here.  Or even an enum.  (Use const)